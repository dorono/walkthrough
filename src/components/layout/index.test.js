import React from 'react';
import {shallow} from 'enzyme';
import {Horizontal, Vertical, VerticalToHorizontal, Box} from './index';

describe('Horizontal Component', () => {
    it('should render without errors', () => {
        shallow(
            <Horizontal />,
        );
    });

    it('should match snapshot', () => {
        expect(shallow(<Horizontal />)).toMatchSnapshot();
    });
});

describe('Vertical Component', () => {
    it('should render without errors', () => {
        shallow(
            <Vertical />,
        );
    });

    it('should match snapshot', () => {
        expect(shallow(<Vertical />)).toMatchSnapshot();
    });
});

describe('VerticalToHorizontal Component', () => {
    const defaultProps = {
        verticalUpTo: 'small',
    };

    it('should render without errors', () => {
        shallow(
            <VerticalToHorizontal {...defaultProps} />,
        );
    });

    it('should match snapshot', () => {
        expect(shallow(<VerticalToHorizontal {...defaultProps} />)).toMatchSnapshot();
    });
});

describe('Box Component', () => {
    it('should render without errors', () => {
        shallow(
            <Box />,
        );
    });

    it('should match snapshot', () => {
        expect(shallow(<Box />)).toMatchSnapshot();
    });
});
