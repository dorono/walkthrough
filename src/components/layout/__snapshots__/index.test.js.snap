// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box Component should match snapshot 1`] = `
<div
  className="box undefined"
/>
`;

exports[`Horizontal Component should match snapshot 1`] = `
<div
  className="horizontal"
/>
`;

exports[`Vertical Component should match snapshot 1`] = `
<div
  className="vertical"
/>
`;

exports[`VerticalToHorizontal Component should match snapshot 1`] = `
<div
  className="verticalToHorizontal verticalUpToSmall"
/>
`;
